NuRadioMC/simulation/simulation2.py

```diff
@@ -644,7 +650,7 @@ def _read_input_neutrino_properties(self):
         self._event_id = self._fin['event_ids'][self._iE]
         self._flavor = self._fin['flavors'][self._iE]
         self._energy = self._fin['energies'][self._iE]
-        self._inttype = self._fin['interaction_type'][self._iE]
+        self._inttype = self._fin['interaction_type'][self._iE].astype('str')
```

Comment:
why is the explicit type conversion necessary?

Reply:
This (and the one above) are to read in eventlists created under python2. Otherwise python3 doesn't know how to handle the strings. I did this so I didn't have to change the eventlists in the unit tests.